### PR TITLE
enhanced the componenet tag.svelte and added a col in presentation fo…

### LINKED ISSUE #76: Include the Presentation Request Status in the presentations table
I would like them to be color coded as tags. I think something like this..

created -> grey or blue
proposed -> orange
requested -> orange
submitted -> yellow

We should also show our role in this table

### PR DESCRIPTION
**Pull request by Vanderscycle**
…r the user to see the status

PR for issue #76

### CODE REVIEW

File: frontend/src/routes/Presentations.svelte [hunk 1]
```diff
@@ -74,21 +110,28 @@ function qrCodeDisplay(id) {
   </div>
   {#if $presentations}
     <DataTable
-      headers="{['', 'Constraints', '']}"
+      headers="{['', 'status', 'Constraints', '']}"
```
Comment (brianorwhatever) [inline frontend/src/routes/Presentations.svelte]:
```suggestion
      headers="{['', 'Status', 'Constraints', '']}"
```

File: frontend/src/routes/Presentations.svelte [hunk 2]
```diff
@@ -26,6 +26,7 @@ onMount(async () => {
   const res = await getPresentations();
   if (res) {
     presentations.set(res);
+    console.log(res);
```
Comment (brianorwhatever) [inline frontend/src/routes/Presentations.svelte]:
```suggestion
```

File: frontend/src/routes/Presentations.svelte [hunk 3]
```diff
@@ -74,21 +110,28 @@ function qrCodeDisplay(id) {
...
-          // },
+          {
+            component: ComponentList,
+            items: [
+              {
+                component: Tag,
+                text: p['status'],
+                bgCol: tailwingBgColorizer(p['status']),
+              },
+              {
```
Comment (brianorwhatever) [inline frontend/src/routes/Presentations.svelte]: Lets put two separate columns here instead of including them both in the same column

Comment (Vanderscycle) [reply frontend/src/routes/Presentations.svelte]: sounds good, will make the changes

Comment (Vanderscycle) [reply frontend/src/routes/Presentations.svelte]: fixed by commit 0c29c48d3e688745ada73befec5125b4dab29a4c

File: frontend/src/routes/Presentations.svelte [hunk 4]
```diff
@@ -109,27 +109,22 @@ function tailwingBgColorizer(value: string): string | void {
   </div>
   {#if $presentations}
     <DataTable
-      headers="{['', 'Status', 'Constraints', '']}"
+      headers="{['', 'role', 'status', 'Constraints', '']}"
```
Comment (brianorwhatever) [inline frontend/src/routes/Presentations.svelte]:
```suggestion
      headers="{['', 'Role', 'Status', 'Constraints', '']}"
```

**Comment by brianorwhatever** [comment]
I accepted my suggested change and am merging in now. Let's try to have proper capitalization on table headers and stuff

**Pull request merged**